.github/ISSUE_TEMPLATE/cran-release.yaml

```diff
@@ -0,0 +1,124 @@
+---
+name: 🎉 CRAN Release
+description: Template for release to CRAN
+title: "[Release]: <version>"
+labels: ["release"]
+assignees:
+  - KlaudiaBB
+  - cicdguy
+  - shajoezhu
```

Comment:
I don't think we need to call @shajoezhu for release of this repo

Reply:
Did you mean me instead of @shajoezhu ?

Also, this .yaml file is propagated from either insightsengineering/.github or insightsengineering/.github-private. If we make changes here, they will still be overwritten during the next propagation. If modifications are necessary, they should be made in those repos.

Reply:
Similar to https://github.com/insightsengineering/teal.data/pull/240/files#r1442001265, let's leave it for now.

Related to https://github.com/insightsengineering/idr-tasks/issues/722